ui/src/kapacitor/components/KapacitorRule.js

```diff
@@ -75,10 +75,13 @@ class KapacitorRule extends Component {
       })
   }
 
-  handleSaveToConfig = () => {
+  handleSaveToConfig = configName => () => {
     const {rule, configLink, router} = this.props
     if (this.validationError()) {
-      router.push(configLink)
+      router.push({
```

Comment:
what do you think about passing through and displaying a hash for the tab that we're on, such that the user would be routed to `http://localhost:8888/sources/1/kapacitors/1/edit#alerta`?

Reply:
much better experience. done.